source/sync/app-builder/tiered-device-sync/index.txt

```diff
@@ -0,0 +1,379 @@
+:noprevnext:
+:orphan:
+
+.. meta::
+   :robots: noindex, nosnippet
+
+.. _tiered-device-sync-guide:
+
+========================
+Tiered Device Sync Guide
+========================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+This guide details how to set up a Tiered Device Sync application while 
+this feature is in pre-alpha. This guide is intentionally not public,
+and is intended for internal MongoDB use.
+
+Create a Device Sync-Enabled App
+--------------------------------
+
+.. procedure::
+
+   .. step:: Create an App Services App
+
+      Create an :ref:`App Services App using the UI, CLI, or Admin API <create-app>`.
+      
+      For a quick proof-of-concept, :ref:`create an app from a 
+      template <create-template-app>`. If you select :guilabel:`Real-time Sync`
+      when you create a template app, this sets up a Device Sync-enabled 
+      task tracker app with a downloadable front-end client. This app comes
+      pre-configured with permissions for the user to write their own Items 
+      to an ``Item`` collection in a ``todo`` database. It also has 
+      :ref:`Development Mode <development-mode>` and 
+      :ref:`email/password authentication <email-password-authentication>` 
+      enabled.
+
+      If you want to define your own permissions and Device Sync settings,
```

Comment:
I wonder if this paragraph would be best left out, because the audience of this procedure might want to just be told exactly what to do?